jscomp/ml/types.mli

```diff
@@ -449,7 +449,7 @@ type label_description =
     lbl_arg: type_expr;                 (* Type of the argument *)
     lbl_mut: mutable_flag;              (* Is this a mutable field? *)
     lbl_pos: int;                       (* Position in block *)
-    lbl_all: label_description array;   (* All the labels in this type *)
+    mutable lbl_all: label_description array;   (* All the labels in this type *)
```

Comment:
This supposedly breaks binary compat, but the editor extension never reads this label.

Reply:
Actually it does not break it.

Reply:
I have been lied to by the machines.

Reply:
Never believe the first thing.
So you kind of break an invariant, but I doubt that it (the invariant) is used anywhere, and definitely not in the editor.